docs/bugs.json

```diff
@@ -1,4 +1,11 @@
 [
+    {
+        "name": "OneOfTwoConstructorsSkipped",
+        "summary": "If a contract has both a new-style constructor (using the constructor keyword) and an old-style constructor (a function with the same name as the contract) at the same time, one of them will be ignored.",
+        "description": "If a contract has both a new-style constructor (using the constructor keyword) and an old-style constructor (a function with the same name as the contract) at the same time, one of them will be ignored. There will be a compiler warning about the old-style constructor, so contracts only using new-style constructors are fine.",
+        "fixed": "0.4.24",
```

Comment:
Should have an introduced line with 0.4.23.

Reply:
fixed.